tests/test_pxssh.py

```diff
@@ -85,6 +85,13 @@ def test_remote_ssh_tunnel_string(self):
         if confirmation_strings!=len(confirmation_array):
             assert False, 'String generated from remote tunneling is incorrect.'
 
+    def test_ssh_config_passing_string(self):
+        ssh = pxssh.pxssh(debug_command_string=True)
+        config_path = '/fakepath/fake/config_file'
```

Comment:
And once the file-exists check is working, you presumably need to create a real named temp file to test with.

It would also be good to have a test with a nonexistent file to check that it errors.

Reply:
yeap.